src/convention/coerce.jl

```diff
@@ -168,3 +168,36 @@ function _check_tight(v::CArr, T, tight)
     end
     return v
 end
+
+
+## Images
+F = Union{Float32, FixedPointNumbers.Normed{UInt8,8}}
```

Comment:
On second thoughts, to cover the cases `Float32` and `FixedPoint`, let's this `F = Real` for now, yes? I can't think of a reason to restrict it. And then we can dump the dependency on FixedPointNumbers.